backend/src/tasks/censysIpv4.ts

```diff
@@ -119,12 +119,24 @@ const downloadPath = async (
 };
 
 export const handler = async (commandOptions: CommandOptions) => {
-  const { chunkNumber, numChunks, organizationId } = commandOptions;
+  const { chunkNumber, organizationId } = commandOptions;
+
+  // Sanitizes numChunks to protect against arbitrarily large numbers
+  const numChucksRawValue = commandOptions.numChunks;
```

Comment:
@Matthew-Grayson moving forward, for future scans or future sanitization we might need to do, it might be good to refactor this code into a helper function (maybe `parseCommandOptions`) as opposed to copy and pasting it across different scans (as it's also here -- https://github.com/cisagov/crossfeed/pull/2123).